diff -Naur oneVPL-cpu-2023.1.2.old/cmake/CompileOptions.cmake oneVPL-cpu-2023.1.2/cmake/CompileOptions.cmake
--- oneVPL-cpu-2023.1.2.old/cmake/CompileOptions.cmake	2023-01-31 07:25:52.964325929 +0100
+++ oneVPL-cpu-2023.1.2/cmake/CompileOptions.cmake	2023-01-31 07:26:52.673301208 +0100
@@ -45,23 +45,7 @@
     set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} /SAFESEH:NO")
   endif()
 else() # compiler options in 'mingw Windows' or 'Linux'
-  # common
-  add_compile_options("-Wall")
-  add_compile_options("-Wformat")
-  add_compile_options("-Wformat-security")
-  add_compile_options("-Werror=format-security")
-  add_definitions("-D_FORTIFY_SOURCE=2")
-
-  # only when warning as error option is enabled in CI test
-  if(ENABLE_WARNING_AS_ERROR)
-    add_compile_options("-Werror")
-  endif()
-
-  if(UNIX)
-    add_compile_options("-fstack-protector-strong")
-    add_link_options("-Wl,-z,relro,-z,now,-z,noexecstack")
-    add_link_options("$<$<CONFIG:Release>:-Wl,--strip-debug>")
-  elseif(WIN32) # mingw in Windows only
+  if(WIN32) # mingw in Windows only
     add_compile_options("-fPIC")
     add_compile_options("-shared")
     # Windows verion of "-z noexecstack"
